tools/bazel.rc

```diff
@@ -2,11 +2,8 @@ build --copt -W
 build --copt -Wall
 build --copt -Wextra
 
-build --cxxopt -W
-build --cxxopt -Wall
```

Comment:
why remove warnings in C++ only?

Reply:
Because `--copt` actually cover C and C++ so they were added twice.
Ref: https://docs.bazel.build/versions/master/user-manual.html#flag--copt